### PR TITLE
Finalize 0.9.5

### LINKED ISSUE #1681: v0.9.5 release planning
Keeping to the quarterly release schedule means releasing 0.9.5 sometime in mid March.  I arbitrarily suggest March 15 for the target release date.

Looking through the [v0.9.5 milestone](https://github.com/pvlib/pvlib-python/milestone/39), these PRs seem like they are close to being ready to merge and are thus candidates for inclusion:

- [x] #1179
- [x] #1183
- [x] #1573
- [x] #1655
- [x] #1682
- [x] #1692
- [x] #1695

Kicking down the road:
- [ ] #1623
- [ ] #1628
- [ ] #1658
- [ ] #1666
- [ ] #1672

@pvlib/pvlib-core please review any PRs that interest you.  I volunteer to solicit reviewers for PRs that don't attract attention on their own over the next couple weeks.

### PR DESCRIPTION
**Pull request by kanderso-nrel**
- [x] Closes #1681
 - [x] Pull request is nearly complete and ready for detailed review.
 - [x] Maintainer: Appropriate GitHub Labels (including `remote-data`) and Milestone are assigned to the Pull Request and linked Issue.

Ready for review

### CODE REVIEW

File: docs/sphinx/source/whatsnew/v0.9.5.rst [hunk 1]
```diff
@@ -95,3 +89,5 @@ Contributors
 * Areeba Turabi (:ghuser:`aturabi`)
 * Mark Campanelli (:ghuser:`markcampanelli`)
 * Taos Transue (:ghuser:`reepoi`)
+* Tim Townsend
```
Comment (kanderso-nrel) [inline docs/sphinx/source/whatsnew/v0.9.5.rst:92]: @cwhanse I wonder if you have an opinion regarding this addition

Comment (cwhanse) [reply docs/sphinx/source/whatsnew/v0.9.5.rst:92]: It seems appropriate to me

**Review by mikofski — APPROVED** [review]
🎉🚀💥

**Comment by kanderso-nrel** [comment]
The sphinx build looks good to me so I'll go ahead with the merge and release.  Thanks to all contributors!

**Pull request merged**